### DISCUSSION TITLE
can we have smooth blinking cursor in kitty like vscode?

### DISCUSSION THREAD

**Discussion by 7549brijesh** [kovidgoyal/kitty#6915] [Q&A]
I was thinking can we customize cursor like vscode
vscode has setting of cursor blinking which has a option called smooth

can we get that smooth blinking cursor in kitty?

I know kittty has the cursor blinking interval but I want cursor to be like vscode smooth blinking.And can this feature can be implemented on kitty as well

For the Smooth typing:
`"editor.cursorSmoothCaretAnimation": true`
For the Smooth cursor Blinking:
`"editor.cursorBlinking": "blink"`

**Comment by hoarse-boy** [comment]
this is an old QnA but maybe this can help people who click on this to find answer like i did.

i haven't used vscode for a longtime, so i don't know the exact animation but maybe this will do

`cursor_blink_interval 0.5 ease-in-out`

put this in kitty config.

**Reply by 7549brijesh** [reply]
thanks it worked any idea on smooth typing?

**Reply by hoarse-boy** [reply]
@7549brijesh is it like vscode or neovide animation when typing? i have searched but found nothing. maybe kitty does not have that.

**Reply by 7549brijesh** [reply]
it is like the vscode if you go to the youtube and search for smooth typing in vscode you will get to know how smooth the cursor moves and typing movement
but i am not sure whether kitty has it or not
